src/cli/uenv.h

```diff
@@ -15,7 +15,7 @@ constexpr int mode_start = 1;
 // constexpr int mode_run = 3;
 
 struct global_settings {
-    bool verbose = false;
+    int verbose = false;
```

Comment:
`int verbose = 0` ?

Reply:
Oops, definitely. Good catch. Fixed in https://github.com/bcumming/uenv2/pull/3/commits/c03807ebee10b106ce4077f17c8422123956668c.